src/Nri/Ui/MediaQuery/V1.elm

```diff
@@ -92,13 +92,14 @@ prefersReducedMotion =
 {-| -}
 notHighContrastMode : List Style -> Style
 notHighContrastMode =
-    withMediaQuery [ "(forced-colors: none)" ]
+    withMediaQuery [ "(forced-colors: none) and (prefers-contrast: no-preference)" ]
```

Comment:
I'm not sure about the prefers-contrast: no-preference 🤔 

If the user has set that they `prefers-contrast: less`, I think we should consider them to _not_ be in high contrast mode. But I think this query would not match, even though it should. Right?

Reply:
Right, I replaced it with `not(prefers-contrast: more)`. Thanks!

Reply:
Accessibility -> Display -> Increase contrast
<img width="659" alt="Screen Shot 2022-10-19 at 13 11 46" src="https://user-images.githubusercontent.com/5647344/196746482-b8e1b8eb-b8bf-40b9-a6e6-dde4a41a999f.png">

Reply:
Awesome, thank you!